src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CSIPReport.kt

```diff
@@ -214,31 +225,63 @@ data class CSIPReport(
   // AUDIT_ADDITIONAL_INFO VARCHAR2(256),
 
   // ---------------------- Decisions & Actions -------------------------/
-  // INV_CONCLUSION VARCHAR2(4000),
-  // INV_OUTCOME VARCHAR2(12),
-  // INV_SIGNED_OFF_BY VARCHAR2(12),
-  // INV_OUTCOME_RECORDED_BY VARCHAR2(100),
-  // INV_OUTCOME_DATE DATE,
-  // INV_NEXT_STEPS VARCHAR2(4000),
-  // INV_OTHER VARCHAR2(4000),
-  // OPEN_CSIP_ALERT VARCHAR2(1) DEFAULT 'N',
-  // INV_NON_ASSOC_UPDATED VARCHAR2(1) DEFAULT 'N',
-  // INV_OBSERVATION_BOOK VARCHAR2(1) DEFAULT 'N',
-  // INV_MOVE VARCHAR2(1) DEFAULT 'N',
-  // INV_REVIEW VARCHAR2(1) DEFAULT 'N',
-  // INV_SERVICE_REFERRAL VARCHAR2(1) DEFAULT 'N',
-  // INV_SIM_REFERRAL VARCHAR2(1) DEFAULT 'N',
+
+  @Column(name = "INV_CONCLUSION")
+  var conclusion: String? = null,
+
+  // is this staff - check
+  @Column(name = "INV_SIGNED_OFF_BY")
+  var signedOffBy: String? = null,
+
+  // is this staff?
+  @Column(name = "INV_OUTCOME_RECORDED_BY")
+  var recordedBy: String? = null,
+
+  @Column(name = "INV_OUTCOME_DATE")
+  var outcomeDate: LocalDate? = null,
+
+  @Column(name = "INV_NEXT_STEPS")
+  var nextSteps: String? = null,
+
+  @Column(name = "INV_OTHER")
+  var otherDetails: String? = null,
+
+  @Column(name = "OPEN_CSIP_ALERT")
+  @Convert(converter = YesNoConverter::class)
+  val openCSIPAlert: Boolean = false,
+  @Column(name = "INV_NON_ASSOC_UPDATED")
+  @Convert(converter = YesNoConverter::class)
+  val nonAssociationsUpdated: Boolean = false,
+  @Column(name = "INV_OBSERVATION_BOOK")
+  @Convert(converter = YesNoConverter::class)
+  val observationBook: Boolean = false,
+  @Column(name = "INV_MOVE")
+  @Convert(converter = YesNoConverter::class)
+  val unitOrCellMove: Boolean = false,
+  @Column(name = "INV_REVIEW")
+  @Convert(converter = YesNoConverter::class)
+  val csraRsraReview: Boolean = false,
```

Comment:
Strange name?

Reply:
I know I copied the name of what was on the screen - might change it now you've said that